UMLEditor/UMLEditor/Views/MainWindow.axaml.cs

```diff
@@ -1348,16 +1138,26 @@ private void Rename_Method_OnClick(object sender, RoutedEventArgs e)
         }
 
         /// <summary>
-        /// Simple function to check if the given input has the correct # arguments for changing needs.
+        ///  Raises and alert with the given parameters
         /// </summary>
-        /// <param name="arguments">String of arguments that we would like to verify</param>
-        /// <param name="numArgsExpected">Number of arguments that we would like 'arguments' to contain</param>
-        /// <returns></returns>
-        private bool isCorrectNumArguments(string arguments, int numArgsExpected)
+        /// <param name="windowTitle">The desired title for the raised alert</param>
+        /// <param name="messageTitle">The desired message title for the raised alert</param>
+        /// <param name="messageBody">The message body for the raise alert</param>
+        /// <param name="alertIcon">The icon you would like present within the alert</param>
+        private void RaiseAlert(string windowTitle, string messageTitle, string messageBody, AlertIcon alertIcon)
         {
-            // Split the arguments string into an array of args, then compare the size of args to the # of arguments we'd like...
-            string[] args = arguments.Split(" ".ToCharArray(), StringSplitOptions.RemoveEmptyEntries);
-            return args.Length == numArgsExpected;
+            // Create and wire up a new modal dialogue to 'AlertPanel' with the parameters being a title and the visible buttons.
+            ModalDialog AlertDialog = ModalDialog.CreateDialog<AlertPanel>(windowTitle, DialogButtons.OKAY);
```

Comment:
At some point, the A in AlertDialog should be lowercased (since it is a local variable).